libs/zephyr-rspress-plugin/src/rspress-plugin/with-zephyr.ts

```diff
@@ -0,0 +1,24 @@
+import path from 'node:path';
+import type { RspressPlugin } from '@rspress/shared';
+import { walkFiles, showFiles } from './utils';
+import { ZeRspressPlugin } from './ze-rspress-plugin';
+
+export function withZephyr(): RspressPlugin {
+  return {
+    name: 'plugin-rspress-ssg',
+
+    async afterBuild({ outDir = 'doc_build' }) {
+      const root = path.resolve(outDir);
+      const files = await walkFiles(root);
+
+      if (files.length === 0) {
+        console.warn('No files found in output directory.');
+        return;
+      }
+
+      await showFiles(root, files);
+
+      await ZeRspressPlugin({ root, files });
+    },
```

Comment:
This different upload process only needed on SSG, right?
Could we have the previous workflow of using rspack plugin here if we don't have ssg enabled?

Reply:
done